docs/knowledge_base/policy_development.md

```diff
@@ -108,3 +138,130 @@ If policy evaluation is successful, the result will contain both the policy outp
 Issues and dependencies that have been suppressed via project preferences are visible in the policy input, but rejections related to those issues or dependencies will not be included in the Markdown report.
```

Comment:
yeah, this is also changing (only issues violating policies at the time of evaluation will be ignored)

Reply:
To be honest, I am not exactly certain on the new behaviors here and would welcome accurate statements. I did not update the development portions, I only consolidated them.

Reply:
Tbh, I am not sure I understand what being "visible in the policy input" means. But not mentioning dependencies in the sentence would make it more accurate.

```suggestion
Issues that have been suppressed via project preferences are visible in the policy input, but the related rejections will not be included in the Markdown report.
```

Reply:
still, a weird sentence

Reply:
You can probably change "visible in the policy input" to "visible to the policy". This is trying to say that the suppressions are applied after the policy is evaluated.

Reply:
that does make more sense to me for sure